spec/mailers/notifications_mailer_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+describe "Mailer" do
+
+  let!(:user) { create(:user) }
+  let!(:current_pack) { create(:pack, title: "current pack", user: user) }
+  let!(:card) { create(:card,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.